tools/flang1/flang1exe/flgdf.h

```diff
@@ -72,4 +72,5 @@ FLG flg = {
     0,     /* accmp */
     "",    /* cmdline */
     FALSE, /* qp */
+    FALSE  /* list-macros */
```

Comment:
Add a comma after the word `FALSE`. This avoids future changes to this line when we add a new field below.